Source/WebCore/workers/service/server/SWScriptStorage.cpp

```diff
@@ -50,11 +50,9 @@ SWScriptStorage::SWScriptStorage(const String& directory)
 
 String SWScriptStorage::sha2Hash(const String& input) const
 {
-    auto crypto = PAL::CryptoDigest::create(PAL::CryptoDigest::Algorithm::SHA_256);
-    crypto->addBytes(m_salt);
-    auto inputUTF8 = input.utf8();
-    crypto->addBytes(inputUTF8.span());
-    return base64URLEncodeToString(crypto->computeHash());
+    auto hash = PAL::CryptoDigest::computeHash(PAL::CryptoDigest::Algorithm::SHA_256, input.utf8().span());
```

Comment:
This change seems unrelated and is unclear if it's an improvement.  If we do it, I think it should be separate.